src/main/java/seedu/address/model/module/ModuleCode.java

```diff
@@ -136,4 +154,14 @@ public boolean hasTutorialClass(TutorialClass tutorialString) {
     public void addTutorialClass(TutorialClass tutorialString) {
         tutorialClasses.add(tutorialString);
     }
+
+    /**
+     * Deletes a tutorial with the given name from the module.
+     * The tutorial has to exist to be used in this function.
+     *
+     * @param tutorialString name of tutorial class to be deleted.
+     */
+    public void deleteTutorialClass(TutorialClass tutorialString) {
```

Comment:
Is there a better naming here? I feel like tutorialString is okay if the type is a String, but in this case if is type TutorialClass that I think we can stick with the convention of tutorialClass.

Reply:
Good point. I'll rename it